keps/sig-apps/2879-ready-pods-job-status/README.md

```diff
@@ -210,7 +211,9 @@ The Job controller will start populating the field again.
 
 ###### Are there any tests for feature enablement/disablement?
 
-Yes, there are tests at unit and [integration] level.
+Yes, there are unit tests (see [link](https://github.com/kubernetes/kubernetes/blob/e8abe1af8dcb36f65ef7aa7135d4664b3db90e89/pkg/controller/job/job_controller_test.go#L236)).
```

Comment:
These are nice feature tests, but (please correct me if I'm wrong), these aren't enablement/disablement tests.

Enablement/disablement tests are tests that effectively change the FG in the middle of the test to see what happens if I enable or disable the feature.

Do you have any tests like these?

Reply:
It's too late now, exactly because what you wrote (the tets won't work after locking FG).

Let's just update the text here to something:
We have feature tests [links to what you linked here] and due to omission we graduated to beta without enablement/disablements tests with just manual test run. With graduation to stable it's too late to add these tests so we're sticking with just manual tests (see below).

Reply:
Thanks, added a note on this.